CONTRIBUTING.md

```diff
@@ -114,6 +114,15 @@ Provide the system information which is not limited to the below:
 
 Please review our [Security Policy](./SECURITY.md). Please follow the instructions outlined in the policy.
 
+## Releasing fetchye
+
+1. Run `npm run lerna:version` locally from your release branch. This would push your release changes(changelog and tags) to the branch on github.
```

Comment:
```suggestion
1. Run `yarn lerna:version` locally from your release branch. This will push your release changes (changelog and tags) to the branch on github.
```